google/cloud/security/common/gcp_api/_base_repository.py

```diff
@@ -0,0 +1,699 @@
+# Copyright 2017 Google Inc.
```

Comment:
Is it worth extracting out the mix-in classes from this file. It's long in the tooth and could provide improved readability.